<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="configTab">
       <attribute name="title">
        <string>Config</string>
       </attribute>
       <widget class="QTableWidget" name="registerReadTable">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>50</y>
          <width>221</width>
          <height>391</height>
         </rect>
        </property>
        <property name="rowCount">
         <number>12</number>
        </property>
        <property name="columnCount">
         <number>1</number>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <row>
         <property name="text">
          <string>ID</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>CONFIG1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>CONFIG2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>LOFF</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>CH1SET</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>CH2SET</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>RLD_SENS</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>LOFF_SENS</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>LOFF_STAT</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>RESP1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>RESP2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>GPIO</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Reg_val</string>
         </property>
        </column>
       </widget>
       <widget class="QComboBox" name="modeComboBox">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>110</y>
          <width>79</width>
          <height>29</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Off</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>On</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Test</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="channel1ComboBox">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>160</y>
          <width>79</width>
          <height>29</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="channel2ComboBox">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>210</y>
          <width>79</width>
          <height>29</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>63</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Mode</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>20</y>
          <width>211</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Register values from device</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Channel 1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Channel 2</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Bitrate</string>
        </property>
       </widget>
       <widget class="QComboBox" name="bitrateComboBox">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>260</y>
          <width>79</width>
          <height>29</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>125</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>250</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>500</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8000</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="saveParamsPushButton">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>330</y>
          <width>88</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Set Params</string>
        </property>
       </widget>
       <widget class="QComboBox" name="setRegisterComboBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>380</y>
          <width>111</width>
          <height>29</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>ID</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CONFIG1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CONFIG2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LOFF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CH1SET</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CH2SET</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RLD_SENS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LOFF_SENS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LOFF_STAT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RESP1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RESP2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GPIO</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>380</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Register name</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>440</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Register value</string>
        </property>
       </widget>
       <widget class="QPushButton" name="saveRegisterPushButton">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>480</y>
          <width>111</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Set Register</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="setRegisterLineEdit">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>430</y>
          <width>113</width>
          <height>29</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="channel1CheckBox">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>160</y>
          <width>21</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QCheckBox" name="channel2CheckBox">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>210</y>
          <width>21</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="modeLineEdit">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>110</y>
          <width>113</width>
          <height>29</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="channel1LineEdit">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>160</y>
          <width>113</width>
          <height>29</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="channel2LineEdit">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>210</y>
          <width>113</width>
          <height>29</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="bitrateLineEdit">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>260</y>
          <width>113</width>
          <height>29</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="RLDCH1PCheckBox">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>300</y>
          <width>21</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>300</y>
          <width>63</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>RLD</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>300</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>CH1P</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>300</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>CH1N</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="RLDCH1NCheckBox">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>300</y>
          <width>21</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>300</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>CH2P</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="RLDCH2PCheckBox">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>300</y>
          <width>21</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>300</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>CH2N</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="RLDCH2NCheckBox">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>300</y>
          <width>21</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="pulseLineEdit_1">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>380</y>
          <width>113</width>
          <height>29</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="pulseLineEdit_2">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>420</y>
          <width>113</width>
          <height>29</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>380</y>
          <width>63</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>HR, CH1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>420</y>
          <width>63</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>HR, CH2</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="logTab">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <widget class="QCheckBox" name="debugCheckBox">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>490</y>
          <width>71</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Debug</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="logOutput">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>800</width>
          <height>500</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>500</height>
         </size>
        </property>
       </widget>
       <widget class="QCheckBox" name="receiverCheckBox">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>450</y>
          <width>101</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Receiver</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="transmitterCheckBox">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>410</y>
          <width>111</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Transmitter</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="plotTab">
       <attribute name="title">
        <string>Plot</string>
       </attribute>
       <layout class="QVBoxLayout" name="plotLayout"/>
      </widget>
      <widget class="QWidget" name="spectrogramTab">
       <attribute name="title">
        <string>Spectrogram</string>
       </attribute>
       <layout class="QVBoxLayout" name="spectrogramLayout"/>
      </widget>
      <widget class="QWidget" name="filterTab">
       <attribute name="title">
        <string>Filters</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="settingsTab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <widget class="QCheckBox" name="darkModeCheckBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>111</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Dark Mode</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="CMV1_CheckBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>21</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>90</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>CM CH1</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="CMV2_CheckBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>21</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>120</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>CM CH2</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1039</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
